README.md

```diff
@@ -1,181 +1,43 @@
-# matchminerCurate
-This application was generated using JHipster 4.13.3, you can find documentation and help at [http://www.jhipster.tech/documentation-archive/v4.13.3](http://www.jhipster.tech/documentation-archive/v4.13.3).
-
-## Development
-
-Before you can build this project, you must install and configure the following dependencies on your machine:
-
-1. [Node.js][]: We use Node to run a development web server and build the project.
-   Depending on your system, you can install Node either from source or as a pre-packaged bundle.
-2. [Yarn][]: We use Yarn to manage Node dependencies.
-   Depending on your system, you can install Yarn either from source or as a pre-packaged bundle.
-
-After installing Node, you should be able to run the following command to install development tools.
-You will only need to run this command when dependencies change in [package.json](package.json).
-
-    yarn install
-
-We use yarn scripts and [Webpack][] as our build system.
-
-
-Run the following commands in two separate terminals to create a blissful development experience where your browser
-auto-refreshes when files change on your hard drive.
-
-    ./mvnw
+# MatchMinerCurate
+MatchMinerCurate is a realtime trial curation platform by using [Clinical Trial Markup Language] (CTML) to label 
+patient-specific genomic and clinical data, and structured eligibility criteria for clinical trials. We use Firebase 
+realtime database to store all trials.
+
+## Install Project
+    npm install
+
+## Prepare Firebase Realtime Database
+1. Go to [Firebase Console] to create a project.
+2. Select `Realtime Database` to store and synchronize data in realtime.
+3. Choose sign-in method and add domain. Go to `Authentication` page and click `Sign-in method`, enable Google as 
+default login method. Of course, you are welcome to try other sign-in providers. Next, add your domain name in 
+`Authorized domains` like "localhost". 
+
+## Prepare configuration files
+1. Prepare files for front-end. 
+    ```
+    cp src/main/webapp/app/environments/environment.example.ts src/main/webapp/app/environments/environment.ts
+    ```
+2. Go to `Authentication` page in Firebase console, click `Web setup` button and Firebase config will pop up. Copy 
+required fields to `environments.ts`.
+    ```html
+    <script src="https://www.gstatic.com/firebasejs/5.1.0/firebase.js"></script>
+    <script>
+      // Initialize Firebase
+      var config = {
```

Comment:
only from line 28-line 33 are needed. It is confusing to put all content in the Web Set up here